react/src/data/wrapper.ts

```diff
@@ -94,21 +100,29 @@ export class ModelsWrapper extends AbstractModelsWrapper<ModelState, unknown> {
     }
 
     hookModel = (modelName: string) => {
-        let selectFn = useSelector;
-        return selectFn(this.modelSelectors.selectModel(modelName));
+        let m = useSelector(this.modelSelectors.selectModel(modelName));
+        if(m === undefined || m === null) {
+            throw new Error("model could not be hooked because it was not found: " + modelName);
+        }
+        return m;
     }
 
     getFieldFromModel(fieldName: string, model: ModelState): unknown {
-        return model[fieldName];
+        let fv = model[fieldName];
```

Comment:
const and single letter

Reply:
About every variable in the program could be a const. If that is what you and @robnagler want I can go around and change them all.